cmd/create.go

```diff
@@ -138,9 +139,16 @@ func create(cmd *cobra.Command, args []string) error {
 
 	if corr.Vars["corral_private_key"] == nil && corr.Vars["corral_public_key"] == nil {
 		logrus.Info("generating ssh keys")
-		privkey, _ := generatePrivateKey(2048)
-		pubkey, _ := generatePublicKey(&privkey.PublicKey)
-		corr.PrivateKey = string(encodePrivateKeyToPEM(privkey))
+		var pubkey []byte
+		if corr.Vars["corral_ssh_key_type"] == "ed25519" {
```

Comment:
If this var isn't set then will it will default to RSA correct?
Also can we make this a const?

Reply:
correct, it defaults to rsa. I'll add a const now